hypothesis-python/scripts/basic-test.sh

```diff
@@ -33,6 +33,8 @@ pip uninstall -y dpcontracts
 
 pip install ".[lark]"
 $PYTEST tests/lark/
+pip install lark-parser==0.7.1
```

Comment:
Does this do the right thing? You're not passing --upgrade and haven't uninstalled the previous version

Reply:
Yes, it uninstalls the current version and installs 0.7.1